caom2-compute/src/main/java/ca/nrc/cadc/caom2/compute/PolygonUtil.java

```diff
@@ -149,8 +150,10 @@ static Polygon getOuterHull(final MultiPolygon poly) {
 
     /**
      * Compute a simple concave outer hull of the specified polygon. This method
-     * removes holes and finds a single concave polygon that includes all the area
-     * of the original (but usually more). This is not a general purpose concave hull
+     * removes holes and finds a single concave polygon that includes all the
+     * area
```

Comment:
Your IDE? I've seen this in a few places...

Reply:
yes, IDE. Fixed

Reply:
IDE also fixed :-)